securedrop/source_app/__init__.py

```diff
@@ -127,14 +127,21 @@ def check_tor2web() -> None:
         # ignore_static here so we only flash a single message warning
         # about Tor2Web, corresponding to the initial page load.
         if 'X-tor2web' in request.headers:
-            flash(Markup(gettext(
-                '<strong>WARNING:&nbsp;</strong> '
-                'You appear to be using Tor2Web. '
-                'This <strong>&nbsp;does not&nbsp;</strong> '
-                'provide anonymity. '
-                '<a href="{url}">Why is this dangerous?</a>')
-                .format(url=url_for('info.tor2web_warning'))),
-                "banner-warning")
+            flash(
+                Markup(
+                    '<strong>{}</strong>&nbsp;{}&nbsp;<a href="{}">{}</a>'.format(
+                        escape(gettext("WARNING!")),
```

Comment:
the all-caps exclamation mark does make it more likely to grab my attention. not sure what the standard is or if we really need one for the first word and formatting used to capture people's attention.

Reply:
Oops, it was originally all caps, but had a colon instead of an exclamation mark. That is a little dramatic. I'll revert it to tone it down.